apps/prairielearn/src/pages/instructorAssessmentInstance/instructorAssessmentInstance.html.ts

```diff
@@ -0,0 +1,705 @@
+                            role="button"
+                            data-toggle="popover"
+                            data-trigger="focus"
+                            data-container="body"
+                            data-html="true"
+                            title="Included in statistics"
+                            data-content="This assessment is included in the calculation of assessment and question statistics"
+                            ><i class="fa fa-question-circle"></i
+                          ></a>
+                        `
+                      : html`
+                          Not included
+                          <a
+                            tabindex="0"
+                            class="btn btn-xs"
+                            role="button"
+                            data-toggle="popover"
+                            data-trigger="focus"
+                            data-container="body"
+                            data-html="true"
+                            title="Not included in statistics"
+                            data-content="This assessment is not included in the calculation of assessment and question statistics because it was created by a course staff member"
+                            ><i class="fa fa-question-circle"></i
+                          ></a>
+                        `}
+                  </td>
+                </tr>
+                <tr>
+                  <th>Date started</th>
+                  <td colspan="2">${assessment_instance_date_formatted}</td>
+                </tr>
+                <tr>
+                  <th>Duration</th>
+                  <td colspan="2">${assessment_instance_duration}</td>
+                </tr>
+              </tbody>
+            </table>
+          </div>
+
+          <div class="card mb-4">
+            <div class="card-header bg-primary text-white">
+              ${resLocals.assessment_instance_label} Questions:
+              ${resLocals.instance_group
+                ? html`${resLocals.instance_group.name} <i class="fas fa-users"></i>`
+                : html`${resLocals.instance_user.name} (${resLocals.instance_user.uid})`}
+            </div>
+
+            <table id="instanceQuestionList" class="table table-sm table-hover">
+              <thead>
+                <tr>
+                  <th>Student question</th>
+                  <th>Instructor question</th>
+                  <th class="text-center">Auto-grading points</th>
+                  <th class="text-center">Manual grading points</th>
+                  <th class="text-center">Awarded points</th>
+                  <th class="text-center" colspan="2">Percentage score</th>
+                  <th></th>
+                </tr>
+              </thead>
+              <tbody>
+                ${instance_questions.map((instance_question, i_instance_question) => {
+                  return html` ${instance_question.start_new_zone && instance_question.zone_title
```

Comment:
Nit (goes along with my next suggestion; may require Prettier reformatting)

```suggestion
                  return html`
                    ${instance_question.start_new_zone && instance_question.zone_title
```

Reply:
Updated formatting locally in [6dfa82c](https://github.com/PrairieLearn/PrairieLearn/pull/9277/commits/6dfa82c90cab62f1a0b4089fc59ee6da989382de).